common/test/acceptance/tests/lms/test_lms_problems.py

```diff
@@ -561,3 +561,33 @@ def get_problem(self):
         </problem>
         """)
         return XBlockFixtureDesc('problem', 'Problem A11Y TEST', data=xml)
+
+class ProblemMetaTest(ProblemsTest):
+    """
+    TestCase Class to verify that the graded variable is passed
+    """
+    def get_problem(self):
+        """
+        Problem structure
+        """
+        xml = dedent("""
+            <problem>
+                <label>Which of the following countries has the largest population?</label>
+                    <multiplechoiceresponse>
+                      <choicegroup type="MultipleChoice">
+                        <choice correct="false">Brazil <choicehint>timely feedback -- explain why an almost correct answer is wrong</choicehint></choice>
+                        <choice correct="false">Germany</choice>
+                        <choice correct="true">Indonesia</choice>
+                        <choice correct="false">Russia</choice>
+                      </choicegroup>
+                    </multiplechoiceresponse>
+            </problem>
+        """)
+        return XBlockFixtureDesc('problem', 'TEST PROBLEM', data=xml, grader_type='Final Exam')
+
+    def test_grader_type_displayed(self):
+        self.courseware_page.visit()
+        problem_page = ProblemPage(self.browser)
+        problem_page.wait_for_element_visibility(problem_page.CSS_PROBLEM_HEADER, 'wait for problem header')
```

Comment:
Should this wait be put into problem_name like you did for problem_progress_graded_value?

Note also the inconsistency between property vs. method call. I'd vote that problem_progress_graded_value should also be a property for consistency.